main.tf

```
@@ -0,0 +1,63 @@
+module "cloudwatch_label" {
+  source  = "cloudposse/label/null"
+  version = "0.25.0"
+
+  delimiter   = "/"
+  label_order = var.label_orders.cloudwatch
+
+  context = module.this.context
+}
+
+resource "aws_cloudwatch_metric_alarm" "success_rate" {
+  count = module.this.enabled ? 1 : 0
+
+  alarm_description   = var.alarm_description
+  alarm_name          = "${module.this.id}-success-rate"
+  datapoints_to_alarm = var.datapoints_to_alarm
+  comparison_operator = "LessThanThreshold"
+  evaluation_periods  = var.evaluation_periods
+  threshold           = var.success_rate_threshold
```

Shouldn't either all or none of the vars used for this alarm be prefixed with `success_rate`? I'd go with no prefix now and later on make it an object variable `var.alarms`
```suggestion
  threshold           = var.threshold
```